static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -385,11 +367,10 @@ CStudioAuthoring.Module.requireModule(
             remove_script_host: false,
             convert_urls: false,
             readonly: _thisControl.readonly, // comes from control props (not xml config)
-            force_br_newlines: forceBRTags, // comes from control props (not xml config)
-            forced_root_block: forceRootBlockPTag, // comes from control props (not xml config)
             remove_trailing_brs: false,
             media_live_embeds: true,
-            contextmenu: !this.enableSpellCheck, // comes from control props (not xml config)
+            contextmenu: !this.enableSpellCheck, // comes from control props (not xml config) TODO: Why is spell check tied to context menu? Access removal.
```

Comment:
@rart https://www.tiny.cloud/docs/tinymce/latest/spell-checking/ according to this doc disabling the contextmenu may be required (though they added that there's an alternative to open the browser context menu).